modules/reactor/src/meshgenerators/ReactorGeometryMeshBuilderBase.C

```diff
@@ -89,3 +92,161 @@ ReactorGeometryMeshBuilderBase::updateElementBlockNameId(
     elem->subdomain_id() = elem_block_id;
   }
 }
+
+void
+ReactorGeometryMeshBuilderBase::printReactorMetadata(const std::string geometry_type,
+                                                     const std::string mg_name,
+                                                     bool first_function_call)
+{
+  if (first_function_call)
+  {
+    Moose::out << "Global metadata defined using Reactor Geometry Mesh Builder:" << std::endl;
```

Comment:
do you have access to _console here?

I m personally not clear on whether _console or Moose::out is better here
@loganharbour ?

Reply:
I can use either _console or Moose::out, not sure which one is preferred

Reply:
I suspect Moose::out would not respond to the console parameters. Does it? (set params in Outputs/Console to check)

Reply:
Hmm, there is console output shown when either _console or Moose::out is used and `Outputs/console=false` in an example file. Not sure if there is a better check I should look at instead

Reply:
let s use console.
there's parameters to the console here:
https://mooseframework.inl.gov/source/outputs/Console.html